### ISSUE TITLE
Issue on noteblock

### ISSUE THREAD

**Issue by starterdragon** [iTXTech/Genisys#2160]
#### Issue description
Note Block doesn't have sounds. It does have it's music notes particles.

#### Steps to reproduce the issue
Just put a note block and use it as a normal noteblock(w/c is I also tried), In my case I used both PlayNoteBlockSong and JukeBoxPE..

#### OS and versions

[10:05:57] [Server thread/INFO]: -----Server information-----
[10:05:57] [Server thread/INFO]: This server is running Genisys-7896cbd (codename "Kyrios")
[10:05:57] [Server thread/INFO]: PHP version: 7.0.12
[10:05:57] [Server thread/INFO]: API: 2.1.0 (iTX API version 1.9.3)
[10:05:57] [Server thread/INFO]: Target client: Minecraft PE v0.16.0.5 alpha (protocol version 91)

```
```

**Comment by BlawoHD** [comment]
This is not the problem the Genisys, this is the problem by the Plugin. Sorry my English is bad

**Comment by starterdragon** [comment]
@BlawoHD I don't think so, just now I did reproduce it on a completely no plugin, there is still no sound

more info:
status
[13:24:13] [Server thread/INFO]: ---- Server status ----
[13:24:13] [Server thread/INFO]: Player count: 0/80
[13:24:13] [Server thread/INFO]: Uptime: 20 seconds
[13:24:13] [Server thread/INFO]: Average TPS: 20 (0%)
[13:24:13] [Server thread/INFO]: Current TPS: 20 (0%)
[13:24:13] [Server thread/INFO]: Network upload: 0 kB/s
[13:24:13] [Server thread/INFO]: Network download: 0 kB/s
[13:24:13] [Server thread/INFO]: Thread count: 7
[13:24:13] [Server thread/INFO]: Main thread memory: 20 MB.
[13:24:13] [Server thread/INFO]: Total memory: 20 MB.
[13:24:13] [Server thread/INFO]: Total virtual memory: 20 MB.
[13:24:13] [Server thread/INFO]: Heap memory: 0 MB.
[13:24:13] [Server thread/INFO]: Maximum memory (system): 20 MB.
[13:24:13] [Server thread/INFO]: World "Main": 0 chunks, 0 entities, 0 tiles. Time 0ms
plugins
[13:24:16] [Server thread/INFO]: Plugins (0):
version
[13:24:18] [Server thread/INFO]: -----Server information-----
[13:24:18] [Server thread/INFO]: This server is running Genisys-7896cbd (codename "Kyrios")
[13:24:18] [Server thread/INFO]: PHP version: 7.0.12
[13:24:18] [Server thread/INFO]: API: 2.1.0 (iTX API version 1.9.3)
[13:24:18] [Server thread/INFO]: Target client: Minecraft PE v0.16.0.5 alpha (protocol version 91)

**Comment by dktapps** [comment]
0.16 changed lots of sound-related things. These have not yet been fixed.

**Comment by Sandertv** [comment]
I have another noteblock related issue. It isn't critical at all so no rush. When there is any block attached to a noteblock, clicking the noteblock will not have an effect. These blocks are for example trapdoors.

**Comment by Goodstuff4UonYT** [comment]
I also have no noteblock sounds.

**Comment by poustmal** [comment]
Pls try to fix it,I had noteblocksong plugin but with this problem,it's useless.

**Comment by dktapps** [comment]
Genisys is no longer under active development. Myself and @JackNoordhuis are the only maintainers at the moment. I have no plans to try and fix this. If anyone manages to fix it themselves feel free to open a pull request and we will review it.

**Comment by Guillaume351** [comment]
Nukkit has a fix for this. Apparently now we have to use a LevelSoundEventPacket to send a note block sound.

**Comment by Sandertv** [comment]
It's all known but Genisys isn't really being worked on anymore.
